store-in-gcs/fake-event.py

```
@@ -0,0 +1,296 @@
+        "name": f"{collection_path}/{document_id}",
+        "fields": firestore_values
+    })
+    
+    # Serialize to protobuf bytes and encode as base64
+    protobuf_bytes = event_data.SerializeToString()
+    return base64.b64encode(protobuf_bytes).decode('utf-8')
+
+
+def generate_curl_command(target_url: str, project_id: str, database_id: str, 
+                         collection_path: str, document_id: str, 
+                         document_fields: Dict[str, Any], use_protobuf: bool = False) -> str:
```

Ruff (0.11.9)

Use `dict` instead of `Dict` for type annotation

Replace with `dict`

(UP006)